app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/Name.cs

```diff
@@ -20,7 +20,7 @@ public class Name : PersonalInfo
         public string Type { get; set; }
 
         [Description("The owner of the name e.g. Applicant, Person Sought, e.t.c")]
-        public OwnerType Owner { get; set; }
+        public OwnerType Owner { get; set; } = OwnerType.NotApplicable;
```

Comment:
We shouldn't put this in the contract. It should be in the mapping or separate definition